lib/groups/help.js

```diff
@@ -3,13 +3,13 @@ const chalk = require('chalk');
 const commandLineUsage = require('command-line-usage');
 
 class HelpGroup {
-    outputHelp(command = null) {
-        if (command !== null) {
-            const commands = require('../utils/commands').info;
+    outputHelp(subject = null, isCommand = true) {
+        if (subject !== null) {
+            const info = isCommand ? require('../utils/commands').info : require('../utils/core-flags').info;
+            const options = isCommand ? info.find(cmd => cmd.name === subject || cmd.alias === subject) : info.find(flg => flg.name === subject.slice(2) || flg.alias === subject.slice(1));
```

Comment:
This one seems a bit hard to read. Maybe use an if/else statement instead.

Reply:
`if/else` would make `options` mutable. Instead used arrow function so as to not lose immutability.

Reply:
Yeah, makes sense :+1: